karapace/schema_registry_apis.py

```diff
@@ -921,6 +930,32 @@ async def subject_versions_list(
                 status=HTTPStatus.NOT_FOUND,
             )
 
+    async def subject_schema_id_registered(
+        self, content_type: str, *, subject: str, schema_id: str, user: User | None = None
+    ) -> None:
```

Comment:
question: I don't know rapu well enough to know this: what happens when we don't enter one of the two branches in the method. What will we return if the method returns `None`?

Reply:
To clarify, I mean: what will the resulting HTTP response be?

Reply:
the result would be a 200 with an empty response body. It's tested in a couple of end to end rest test (I agree isn't clear and isn't beautiful)

Reply:
Ok, I think it's fine, more an issue with the framework. I would consider responding with HTTP 204, but that's not a blocker IMO.